clientv3/integration/txn_test.go

```diff
@@ -74,7 +74,7 @@ func TestTxnWriteFail(t *testing.T) {
 
 	dialTimeout := 5 * time.Second
 	select {
-	case <-time.After(2*dialTimeout + time.Second):
+	case <-time.After(3*dialTimeout + time.Second):
```

Comment:
Actually, we don't have control in the dial in this case I think. (we use `context.TODO`). It looks like gRPC sets its own timeout when we don't pass timeout https://github.com/grpc/grpc-go/blob/master/clientconn.go#L452-L459.